dali/kernels/imgproc/resample/resampling_setup.cc

```diff
@@ -98,80 +100,196 @@ SeparableResamplingSetup<spatial_ndim>::ComputeScaleAndROI(
   return roi;
 }
 
+template <int ndim>
+struct ProcessingOrderCalculator {
+  ProcessingOrderCalculator(ivec<ndim> in_size, ivec<ndim> out_size, ivec<ndim> filter_support)
+  : in_size(in_size), out_size(out_size), filter_support(filter_support) {
+  }
+
+  ProcessingOrder<ndim> operator()() {
+    for (int i = 0; i < ndim; i++)
+      best_order[i] = i;
+    min_cost = 1e+30f;
```

Comment:
numeric_limits ?

Reply:
Meh. Because `std::numeric_limits<float>::max()` is so much easier to write and read when you just want a "big enough" number.

Reply:
I just though that this simple way leads to the situation when one day big enough number is not big enough anymore...